OrbitGl/ThreadTrack.h

```diff
@@ -58,6 +58,7 @@ class ThreadTrack : public Track {
 
   int32_t GetThreadId() const { return thread_id_; }
   bool IsCollapsable() const override { return depth_ > 1; }
+  virtual float GetYFromDepth(uint32_t depth);
```

Comment:
Should this be overridden in SchedulerTrack? If not, does it need to be virtual?

Reply:
No! :)